app/views/users/show.html.erb

```diff
@@ -1,2 +1,31 @@
-<h1> Here is an author with her/his corresponding bio and his/her posts
-</h1>
+<div class="user-posts-wrapper">
+  <h1 class="header-text"> Here is an author with her/his corresponding bio and his/her posts </h1>
+  <div class="user-posts">
+    <div class="author-info user-link-container">
+        <img class="post-box" src ="/box-post.png" alt="<%= @user.name %>'s profile picture">
+        <div class="user-link-wrapper">
+          <p class="username-link"> <%= link_to @user.name, user_path(@user) %> </p>
+          <p class="posts-counter">  Number of posts: <%= @user.posts_counter %> </p>
+        </div>
+    </div>
```

Comment:
- [ ] @amare1990 great job on your views it would be nice if you ensured the layout matches the design given currently the comments and likes counter are not rightly placed kindly check out the comparison below:

Design Given         |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/216413027-ed7fbb10-f407-411f-9b86-84f4b27954fa.png)  |  ![image](https://user-images.githubusercontent.com/61408860/216413538-b80771a1-019c-435e-9add-d994b1d3e18f.png)

Reply:
Well resolved! 👍🏾 

![image](https://user-images.githubusercontent.com/38088818/216451961-60e07276-c19f-44e9-a3a6-3d14bcb57f9f.png)